### PR TITLE
Texture hacks

### PR DESCRIPTION
**Pull request by JayFoxRox**
Needs some more work still.. feel free to continue this for me. I'll be busy with other stuff the next days

Anakins pod

### CODE REVIEW

**Comment by JayFoxRox** [comment]
~~Textures are actually upside down I believe.~~
~~This is a problem with the out_textureblock.py script!~~

Fixed since the screenshot was taken.

**Review by Olganix — APPROVED** [review]

**Review by Olganix — APPROVED** [review]

File: src/Swr_Model.cpp [hunk 1]
```diff
@@ -623,8 +623,8 @@ void Collada::addImage(string name, string filename)
 {
 	TiXmlElement* node_image = new TiXmlElement("image");
 	node_library_images->LinkEndChild(node_image);
-	node_image->SetAttribute("id", name);
-	node_image->SetAttribute("name", name);
+	node_image->SetAttribute("id", name + "_texture");
+	node_image->SetAttribute("name", name + "_texture");
```
Comment (JayFoxRox) [inline src/Swr_Model.cpp:627]: Might be bad now? Have to review other changes by @Olganix

Comment (JayFoxRox) [reply src/Swr_Model.cpp:627]: Change looks correct!

**Comment by JayFoxRox** [comment]
Not so hacky anymore. Upstreamed.

**Pull request closed without merging**